docs/command-reference.md

```diff
@@ -449,7 +449,7 @@ Usage: `nerdctl ps [OPTIONS]`
 
 Flags:
 
-- :whale: `-a, --all`: Show all containers (default shows just running)
+- :whale: `-a, --all`: Show all containers (default shows just running). When specifying the option '-f' with 'status', it filters through all containers
```

Comment:
I would rather change the `-f` flag line 463

`:whale: -f, --filter: Filter all containers based on given conditions`